sources/packages/api/src/services/education-program-offering/education-program-offering.service.models.ts

```diff
@@ -0,0 +1,14 @@
+export class EducationProgramOfferingModel {
+  id: number;
+  name: string;
+  studyStartDate: Date;
+  studyEndDate: Date;
+  offeringDelivered: string;
+  get studyDates(): string {
+    if (this.studyStartDate === null) {
+      return "No program dates";
+    } else {
+      return `${this.studyStartDate.toLocaleDateString()} - ${this.studyEndDate.toLocaleDateString()}`;
```

Comment:
I would recommend the use of a library like dayjs (already in the solution) to format the dates in the expected format.

Reply:
done